cfs_config_util/bin/vcs_creds_helper.py

```diff
@@ -31,7 +31,7 @@
 import base64
 import sys
 
-from cfs_config_util.apiclient import load_kube_api
+from csm_api_client.k8s import load_kube_api
```

Comment:
Does anything need this file anymore? Aren't VCS creds handled in [python-csm-api-client](https://github.com/Cray-HPE/python-csm-api-client/blob/61ed8d46a73db1488fd36ae7c456a8d10f52ced3/csm_api_client/service/vcs.py#L56-L76) now?

Reply:
Yes, that's correct. I removed this script and the entrypoint in `setup.py`.